src/assets/i18n/fr.json5

```diff
@@ -6108,7 +6108,7 @@
   "submission.sections.general.save_success_notice": "Soumission sauvegardée avec succès.",
 
   // "submission.sections.general.search-collection": "Search for a collection",
-  "submission.sections.general.search-collection": "Rechercher une collection",
+  "submission.sections.general.search-collection": "Rechercher un(e) collection",
```

Comment:
As this label applies only to Collection, it is better to simply use "une" as "collection" in french is always feminine.